File: extract_wards.js
const fs = require('fs');
const content = fs.readFileSync('src/app/components/registration/registration.ts', 'utf8');
const match = content.match(/wards = signal<any>\(\[\s*([\s\S]*?)\s*\]\);/);
if (match) {
    fs.writeFileSync('src/app/shared/wards.ts', 'export const WARDS = [\n' + match[1] + '\n];\n');
    console.log('Success');
} else {
    console.log('Not found');
}

File: extract_wards.py
import re

with open(r'c:\Users\Emperor\Desktop\New stuff\Chagua\Uchaguzi-Frontend\src\app\components\registration\registration.ts', 'r', encoding='utf-8') as f:
    txt = f.read()

match = re.search(r'wards = signal<any>\(\[\s*(.*?)\s*\]\);', txt, re.DOTALL)
if match:
    with open(r'c:\Users\Emperor\Desktop\New stuff\Chagua\Uchaguzi-Frontend\src\app\shared\wards.ts', 'w', encoding='utf-8') as out:
        out.write('export const WARDS = [\n' + match.group(1) + '\n];\n')
    print('Successfully created wards.ts')
else:
    print('Could not find wards in registration.ts')
